openquake/commonlib/reportwriter.py

```diff
@@ -35,13 +35,15 @@ class ReportWriter(object):
         source_data_transfer='Expected data transfer for the sources',
         avglosses_data_transfer='Estimated data transfer for the avglosses',
         exposure_info='Exposure model',
+        short_source_info='Slowest sources',
+        performance='Slowest operations',
     )
 
     def __init__(self, dstore):
         self.dstore = dstore
         self.oq = oq = OqParam.from_(dstore.attrs)
         self.text = oq.description + '\n' + '=' * len(oq.description)
-        sitecol_size = humansize(len(dstore.hdf5['sitecol'].value))
+        sitecol_size = humansize(len(parallel.Pickled(dstore['sitecol'])))
```

Comment:
Please add a comment, specifying that pickling is currently the right way to estimate the data transfer, but we could change it if we transfer the sitecol as an array